From 38ea7e821bce1cbf3c8fe4dc044a809301a77f82 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@fedoraproject.org>
Date: Tue, 22 Sep 2020 12:01:17 +0100
Subject: [PATCH] Include devfreq drivers in initrd

Some SoCs now have drivers that user devfreq in early init and fail
if the drivers are missing so make sure we have them in the initrd.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 modules.d/90kernel-modules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 764675eb..3ed749c7 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -65,6 +65,7 @@ installkernel() {
             _blockfuncs+='|dw_mc_probe|dw_mci_pltfm_register'
             instmods \
                 "=drivers/clk" \
+                "=drivers/devfreq" \
                 "=drivers/dma" \
                 "=drivers/extcon" \
                 "=drivers/gpio" \

